src/tests/ftest/control/dmg_pool_query_test.py

```diff
@@ -0,0 +1,161 @@
+
+        # Parse output
+        d_info = get_pool_query_info(dmg_out.stdout)
+        self.log.info("dmg values found: {}".format(d_info))
+
+        e_info = self.params.get("exp_vals", "/run/*")
+        self.log.info("Expected values are: {}".format(e_info))
+
+        # Verify
+        if d_info != e_info:
+            self.fail("dmg pool query expected output: {}".format(e_info))
+
+    def test_pool_query_inputs(self):
+        """
+        JIRA ID: DAOS-2976
+        Test Description: Test basic dmg functionality to query pool info on
+        the system. Verify the inputs that can be provided to 'query --pool'
+        argument of the dmg pool subcommand.
+        :avocado: tags=all,tiny,pr,hw,dmg,pool_query,basic
+        """
+        # Get test UUID
+        uuid = self.params.get("uuid", '/run/pool_uuids/*/')
+        self.log.info("Using test UUID: %s", uuid[0])
+
+        self.log.info("Running dmg pool query")
+        dmg_out = pool_query(self.bin, self.host_p, uuid[0])
+
+        # Verify
+        self.log.info("Test is expected to finish with: %s", uuid[1])
+        if dmg_out is None:
+            exception = 2
+        else:
+            if dmg_out.exit_status == 0:
+                exception = None
+            else:
+                exception = 1
+
+        if uuid[1] == "FAIL" and exception is None:
+            self.log.error("Command was expected to fail")
+            self.fail("Test failed, dmg pool query command output: {}".format(
+                dmg_out.stdout))
+        elif uuid[1] == "PASS" and exception is not None:
+            self.log.error("Command was expected to pass")
+            if exception == 2:
+                self.fail("Test failed, dmg command failed while executing.")
+            if exception == 1:
+                self.fail("Test failed, dmg command failed with {}".format(
+                    dmg_out.stdout))
+
+    def test_pool_query_ior(self):
+        """
+        JIRA ID: DAOS-2976
+        Test Description: Test that pool query command will properly and
+        accurately show the size changes once there is content in the pool.
+        :avocado: tags=all,tiny,pr,hw,dmg,pool_query,basic
+        """
+        # Store orignal pool info and run ior
+        self.log.info("Getting pool info before writting data")
+        out_before_ior = pool_query(self.bin, self.host_p, self.pool.pool.uuid)
+        self.run_ior_with_pool()
+
+        # Check pool written data
+        self.log.info("Getting pool info after ior run")
+        out_after_ior = pool_query(self.bin, self.host_p, self.pool.pool.uuid)
+
+        # Parse output of dmg command before running ior
+        orig_pool_info = get_pool_query_info(out_before_ior.stdout)
+        self.log.info("dmg values found: {}".format(orig_pool_info))
+
+        # Parse output of dmg command after running ior
+        curr_pool_info = get_pool_query_info(out_after_ior.stdout)
+        self.log.info("dmg values found: {}".format(curr_pool_info))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments